webapp/apps/dynamic/tests/test_behavioral.py

```diff
@@ -148,12 +148,15 @@ def test_get_not_avail_page_renders(self, start_year, start_year_is_none):
                                         UrlModel=DynamicBehaviorOutputUrl)
 
         model = unique_url.unique_inputs
-        model.raw_input_fields = None
-        model.input_fields = None
-        model.deprecated_fields = None
+        model.raw_input_fields = {}
+        model.input_fields = {}
+        model.deprecated_fields = {}
         model.tax_result = "unrenderable"
```

Comment:
You can remove the code that changes `raw_input_fields`, `input_fields`, and `deprecated_fields`. There's a similar test in `taxbrain/tests/test_views` that has these lines of code. You can remove those, too.